### PR TITLE
Add Lady Bug.

### PR DESCRIPTION
**Pull request by Taranchul**

### CODE REVIEW

File: bezel-cfgs/ladybug.cfg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# 1941 Counter Attack Overlay for Retroarch with 4:3 Display
```
Comment (svera) [inline bezel-cfgs/ladybug.cfg]: This overlay is not for 1941, isn't it? :)

File: cfgs/ladybug.cfg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+input_overlay = "/opt/retropie/emulators/retroarch/overlays/arcade-bezels/ladybug.cfg"
+input_overlay_enable = "true"
+input_overlay_opacity = "0.900000"
```
Comment (svera) [inline cfgs/ladybug.cfg]: I would put the opacity as 1.000000 to make it completely opaque, although it won't make any difference

**Review by svera — COMMENTED** [review]
Thank you for your contribution @Taranchul ! Just a couple of minor issues though

File: bezel-cfgs/ladybug.cfg (hunk 1 above)
Comment (Taranchul) [reply bezel-cfgs/ladybug.cfg]: You're right, I forgot to change that. But I'll make a minor update to the artwork anyway, so stay tuned.

File: cfgs/ladybug.cfg (hunk 2 above)
Comment (Taranchul) [reply cfgs/ladybug.cfg]: I'll look into that, too. Thanks!

File: bezel-cfgs/ladybug.cfg (hunk 1 above)
Comment (Taranchul) [reply bezel-cfgs/ladybug.cfg]: Done, see second commit.

File: cfgs/ladybug.cfg (hunk 2 above)
Comment (Taranchul) [reply cfgs/ladybug.cfg]: Done, see second commit.

**Review by svera — APPROVED** [review]
Lovely, thank you!

**Pull request merged**